ksql-engine/src/main/java/io/confluent/ksql/function/udaf/topk/TopkAggFunctionDeterminer.java

```diff
@@ -19,35 +19,63 @@
 import org.apache.kafka.connect.data.Schema;
 
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 import io.confluent.ksql.function.KsqlAggFunctionDeterminer;
 import io.confluent.ksql.function.KsqlAggregateFunction;
 import io.confluent.ksql.util.KsqlException;
+import org.apache.kafka.connect.data.SchemaBuilder;
 
 public class TopkAggFunctionDeterminer extends KsqlAggFunctionDeterminer {
```

Comment:
@hjafarpour what do you think if I rename `TopkAggFunctionDeterminer` to `TopKAggFunctionFactory`? I think that factory better communicates functionality of this class.

Reply:
@satybald sounds good. please go ahead and rename it along with the other ones.